mllib/src/test/scala/org/apache/spark/ml/feature/NormalizerSuite.scala

```diff
@@ -105,6 +105,23 @@ class NormalizerSuite extends SparkFunSuite with MLlibTestSparkContext with Defa
     assertValues(result, l1Normalized)
   }
 
+  test("Normalization should throw adequate exception on input type mismatch") {
+    val data = Seq(Tuple1("string value"))
+
+    val df = sqlContext.createDataFrame(data).toDF("features")
+
+    val normalizer = new Normalizer()
+      .setInputCol("features")
+      .setOutputCol("polyFeatures")
```

Comment:
Fixed: `normalized_features` is the column name used in other tests in NormalizerSuite.scala,
thanks!